linkcheck/__init__.py

```diff
@@ -22,14 +22,6 @@
 if sys.version_info < (3, 5, 0, 'final', 0):
     import platform
     raise SystemExit("This program requires Python 3.5.0 or later instead of %s." % platform.python_version())
-# require a reasonably recent requests module: 2.4.0 from 2014-08-29
-import requests
```

Comment:
it looks like this completely removes the import - it was unused?

Reply:
Not in this file, just there to check the version.